neptune/internal/integration/keras.py

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+_integrated_with_keras = False
+
+
+def integrate_with_keras():
+    global _integrated_with_keras  # pylint:disable=global-statement
+
+    if _integrated_with_keras:
+        return
+
+    _integrate_with_keras()
+
+    _integrated_with_keras = True
+
+
+def _integrate_with_keras():
+    try:
+        import keras
+    except ImportError:
+        raise ValueError('Requested integration with keras while keras is not installed.')
+
+    def get_current_global_experiment():
+        pass  # TODO get current experiment from global state
```

Comment:
A leftover?

Reply:
After rebasing this will be replaced by a dynamic 'global' experiment getter